components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/StandaloneJobOrchestrator.cs

```diff
@@ -864,6 +874,11 @@ IJsonSerializer jsonSerializer
                                 found = true;
                             }
                             matchingGroups.Add(entry);
+
+                            // refresh the Tag if a new one is provided (not exactly necessary, but might be useful for debugging)
+                            if (!string.IsNullOrEmpty(request.Tag)) {
```

Comment:
Hm, on one hand I want to ask you to move refresh of `Tag` field under `if (entry.HasSameDataSet(request))` condition on line `872` so that its application is consistent across direct method handlers. Otherwise endpoints not intended by the call will be affected. But on the other hand those entries will be removed from `_publishedNodesEntries` with the `RemoveEntriesWithoutNodes()` method call on line `891`. So I am wondering if we need to do any update of `Tag` field in `UnpublishAllNodesAsync()` method at all as endpoints matching the requested one will be removed and they would be the ones to be updated.

Reply:
I had same thought. I will remove it